packages/server/graphql/types/TeamPromptResponse.ts

```diff
@@ -0,0 +1,55 @@
+import {GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import Team from './Team'
+
+const TeamPromptResponse: GraphQLObjectType = new GraphQLObjectType<any, GQLContext>({
+  name: 'TeamPromptResponse',
+  description: 'A response of a single team member in a team prompt',
+  isTypeOf: ({status}) => !!status,
```

Comment:
Where does this `status` come from?

Reply:
that wasn't really needed, it's removed now